topics/admin/tutorials/job-destinations/tutorial.md

```diff
@@ -270,7 +270,7 @@ We want our tool to run with more than one core. To do this, we need to instruct
 >    -        env:
 >    -        - name: LC_ALL
 >    -          value: C
->    -        - name: SINGULARITY_CACHEDIR
+>    -        - name: APPTAINER_CACHEDIR
```

Comment:
These does not seem a spelling mistake. Can you please explain why are you making this change?

Reply:
I guess I should have made a separate PR for this. Throughout the rest of the training material, APPTAINER_ is used (and partly in this tutorial), so I assume it was just a mistake to not previously change this from SINGULARITY_.

Reply:
If singularity is not used at all then the corresponding value on the next line also does not make sense and more changes are required I believe

Reply:
Some of these variables are interpreted by Galaxy, and must be `SINGULARITY` (mostly job conf, e.g. `singularity_enabled`) 

Others are read by apptainer and can safely be `APPTAINER`. `TMPDIR` is one of those that isn't read by galaxy afaict (grepping release_23.1`)

```
$ ag CACHEDIR
doc/source/admin/container_resolvers.rst
169:For singularity, admins should also take care of the ``APPTAINER_CACHEDIR``
```

cachedir it seems is too.

Reply:
Yeah, and in the Apptainer tutorial, both `APPTAINER_CACHEDIR` and `APPTAINER_TMPDIR` are used.